rest-api/tools/csv_importer.py

```diff
@@ -81,8 +81,21 @@ def _update_entity(self, entity, existing_entity, session, dry_run):
       logging.info('Not updating %s.', new_dict[self.external_id_field])
```

Comment:
FYI: I'm also considering dropping this line since it's pretty spammy and IMO not very useful as you can infer skips by looking at the "total_skipped" at the end. Anyone feel strongly about this?

Reply:
Seems fine to me. @Seeker1911, WDYT?